core/frontend/src/components/parameter-editor/InlineParameterEditor.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <v-form
+    ref="form"
+    v-model="is_form_valid"
+    @submit.prevent="saveEditedParam()"
+  >
+    <template v-if="!custom_input && param?.bitmask">
+      <v-checkbox
+        v-for="(key, keyvalue) in param?.bitmask"
+        :key="keyvalue"
+        v-model="selected_bitflags"
+        dense
+        hide-details
+        :loading="waiting_for_param_update"
+        :label="key"
+        :value="2 ** keyvalue"
+      />
+    </template>
+    <v-autocomplete
+      v-else-if="!custom_input && Object.entries(param?.options ?? []).length > 10"
+      v-model.number="new_value"
+      variant="solo"
+      :items="as_select_items"
+    />
+    <v-select
+      v-else-if="!custom_input && param?.options"
+      v-model.number="new_value"
+      dense
+      :items="as_select_items"
+      :rules="[() => true]"
```

Comment:
what ?

Reply:
also old code, doesnt's seem like it is required. removed